packages/edit-site/src/components/revisions/index.js

```diff
@@ -88,6 +103,10 @@ function Revisions( { userConfig, blocks } ) {
 						settings={ settings }
 					>
 						<BlockList renderAppender={ false } />
+						<RevisionStyles
```

Comment:
At this point the overrides from the useBlockProps hooks are in the store.

Reply:
But <EditorStyles> normally listens for changes to the store? Why doesn't that work here?